packages/utilities/src/utilities.ts

```diff
@@ -224,7 +224,7 @@ const FORBIDDEN_USERNAMES_REGEXPS = [
     'covid', 'covid19', 'matfyz', 'ideas', 'public-actors', 'resources', 'partners', 'affiliate',
     'industries', 'web-scraping', 'custom-solutions', 'solution-provider', 'alternatives', 'platform',
     'freelancers', 'freelancer', 'partner', 'preview', 'templates', 'data-for-generative-ai',
-    'discord',
+    'discord', 'praguecrawl', 'prague-crawl', 'bob',
```

Comment:
Now I have [this](https://youtu.be/LyUIJIdwEuA?si=tZqTMGIsHc_yLUlR) song stuck in my head🙂